<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TourPropertyUI</class>
 <widget class="QWidget" name="TourPropertyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游览管理</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="selectPath">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览..</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tourPath">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>游览路径：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTour">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>loadTour</zorder>
  <zorder>close</zorder>
  <zorder>loadTour</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectPath</sender>
   <signal>clicked()</signal>
   <receiver>TourPropertyUI</receiver>
   <slot>slotSelectPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadTour</sender>
   <signal>clicked()</signal>
   <receiver>TourPropertyUI</receiver>
   <slot>slotLoadTour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>TourPropertyUI</receiver>
   <slot>slotClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSelectPath()</slot>
  <slot>slotLoadTour()</slot>
  <slot>slotUnloadTour()</slot>
  <slot>slotClose()</slot>
 </slots>
</ui>
